OrbitGl/SchedulerTrack.cpp

```diff
@@ -18,7 +18,7 @@ using orbit_client_protos::TimerInfo;
 const Color kInactiveColor(100, 100, 100, 255);
 const Color kSelectionColor(0, 128, 255, 255);
 
-SchedulerTrack::SchedulerTrack(TimeGraph* time_graph) : TimerTrack(time_graph) { SetPinned(true); }
+SchedulerTrack::SchedulerTrack(TimeGraph* time_graph) : TimerTrack(time_graph) { SetPinned(false); }
```

Comment:
Is this needed for the track dragging?

Reply:
There are a few small issues.

It isn't needed, but I'm not sure how we want to implement pinning, maybe we can make something different but didn't think too much about that. I'm trying to push moving tracks to 1.56, and I don't have priority on Pinning for now.

Reply:
Maybe we should leave it in for now, and disable in a separate PR? @reichlfl ?

Reply:
I think it's fine to just disable the pinning of this track. It's independent of the track ordering, and the SchedulerTrack is only pinned to test the pinning-feature (when the feature flag is enabled), so there's no harm in changing this. Agree with Pablo, this way we can safely remove the feature flag since the user has no way of enabling track pinning through the UI.